lib/fastlane/plugin/wpmreleasetoolkit/actions/common/update_assigned_milestone_action.rb

```diff
@@ -74,22 +74,22 @@ def self.available_options
                                        description: 'The remote path of the GH repository on which we work',
                                        optional: false,
                                        type: String),
-          FastlaneCore::ConfigItem.new(key: :pr_numbers,
-                                       description: 'The PR numbers to update the milestone of',
+          FastlaneCore::ConfigItem.new(key: :numbers,
+                                       description: 'The PR and/or issue numbers to update the milestone of',
                                        optional: true,
                                        type: Array,
                                        conflicting_options: [:from_milestone]),
           FastlaneCore::ConfigItem.new(key: :from_milestone,
-                                       description: 'The version (milestone title\'s start) for which we want to update all open PRs of to a new milestone',
+                                       description: 'The version (milestone title\'s start) for which we want to update all open PRs and issues of to a new milestone',
```

Comment:
Small suggestion trying to make it a bit cleaner:
```suggestion
                                       description: 'The version (start of the milestone title) for which we intend to update all open PRs and issues to a new milestone.',
```